app/src/main/java/com/keylesspalace/tusky/components/notifications/FollowViewHolder.kt

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2023 Tusky Contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>.
+ */
+
+package com.keylesspalace.tusky.components.notifications
+
+import androidx.recyclerview.widget.RecyclerView
+import com.keylesspalace.tusky.R
+import com.keylesspalace.tusky.databinding.ItemFollowBinding
+import com.keylesspalace.tusky.entity.Notification
+import com.keylesspalace.tusky.entity.TimelineAccount
+import com.keylesspalace.tusky.util.StatusDisplayOptions
+import com.keylesspalace.tusky.util.emojify
+import com.keylesspalace.tusky.util.loadAvatar
+import com.keylesspalace.tusky.util.unicodeWrap
+import com.keylesspalace.tusky.viewdata.NotificationViewData
+
+class FollowViewHolder(
+    private val binding: ItemFollowBinding,
+    private val notificationActionListener: NotificationActionListener,
+) : NotificationsPagingAdapter.ViewHolder, RecyclerView.ViewHolder(binding.root) {
+    private val avatarRadius42dp = itemView.context.resources.getDimensionPixelSize(
+        R.dimen.avatar_radius_42dp
+    )
+
+    override fun bind(
+        viewData: NotificationViewData,
+        payloads: List<*>?,
+        statusDisplayOptions: StatusDisplayOptions
+    ) {
+        // Skip updates with payloads. That indicates a timestamp update, and
+        // this view does not have timestamps.
+        if (!payloads.isNullOrEmpty()) return
```

Comment:
Should we have a more clear condition here?

("List.isNullOrEmpty" can only understood with the comment. It would be easier if it were something like "isTimestampUpdate".)

Reply:
Possibly, but this is based on existing code, so if it's good enough for that code it's good enough for this code.

A future cleanup can adjust if needed, otherwise this PR becomes an opportunity for everyone to continue to suggest more small changes.

The ongoing maintenance burden of this PR is high -- there are merge conflicts on almost every change that gets merged in to `develop`, so I prefer to get this in sooner.